src/dotnet-grpc/dotnet-grpc.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Authors>The gRPC Authors</Authors>
+    <Copyright>Copyright 2019 The gRPC Authors</Copyright>
+    <Description>Command line tool for gRPC projects</Description>
+    <PackageIconUrl>https://github.com/grpc/grpc.github.io/raw/master/img/grpc_square_reverse_4x.png</PackageIconUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/grpc/grpc-dotnet</PackageProjectUrl>
+    <PackageTags>gRPC RPC CLI</PackageTags>
+    <VersionPrefix>$(GrpcDotnetVersion)</VersionPrefix>
+
+    <!-- Ignore missing XML documentation -->
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <OutputType>exe</OutputType>
+    <!--<PackAsTool>true</PackAsTool>-->
```

Comment:
This need to be uncommented to ship. Currently disabled to allow for easier debugging.